common/js/src/popup-window/server.js

```diff
@@ -65,7 +65,8 @@ GSC.PopupWindow.Server.createWindow = function(
 
   logger.fine(
       'Creating a popup window with url="' + url + '", options=' +
-      GSC.DebugDump.debugDump(createWindowOptions));
+      GSC.DebugDump.debugDump(createWindowOptions) + ', data=' +
```

Comment:
Returned this logging back, as the recent changes made the debug dumping functions more robust.